submissions/yanyshpolska/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="/img/favicon.png" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Hooli-style Popup</title>
+  </head>
+  <body>
+    <header>
+      <nav class="menu__items">
```

Comment:
for plural naming better use `list` word for example, because item and items looks pretty the same and you need to pay much attention to see the difference